doc/man1/ciphers.pod

```diff
@@ -346,7 +346,7 @@ Cipher suites using SHA1.
 
 =item B<SHA256>, B<SHA384>
 
-Ciphersuites using SHA256 or SHA384.
+Cipher suites using SHA256 or SHA384.
```

Comment:
All the documentation should be consistent now.

I've not touched the NEWS, INSTALL or CHANGES files at the top level.  The rest of the references to _ciphersuite_ are in code and tests.